src/lif_cell_description.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+// Model parameteres of leaky integrate and fire neuron model.
+struct lif_cell_description {
```

Comment:
I see your point. However, the parameter V_m describes the initial membrane potential, so I think it should still be part of the frontend description. I sometimes see scientists like to put that the neurons have initial membrane potentials sampled from some distribution, so that not all the neurons in the network are the same (thus creating a more robust setting). However, I should definitely be more precise then and write in the comment that V_m represents the initial potential.